app/assets/stylesheets/components/_post.scss

```diff
@@ -80,6 +83,64 @@ $Post-inlineCodeBgColor: rgba(0, 0, 0, 0.04);
     }
   }
 
+  .Table {
```

Comment:
If I do that, won't line 27 be breaking Suit? There would be shared knowledge among two different components.